primeSearch.js

```diff
@@ -221,7 +225,11 @@ export async function findPrime (original, sophie = false) {
         const tests = generateTests(tested, keyFrame, simultaneous)
 
         // Turn the tests into `openssl prime` processes, and wait for them to complete.
-        const result = await Promise.all(tests.map(i => execPromise(`openssl prime ${i}`)))
+        const result = await Promise.all(
+            tests
+                .filter(i => isPossiblyPrime(i))
```

Comment:
Hey, perhaps instead of implementing this here (while this is cleaner), we might be able to move this into the `generateTests` loop. 

```js
const candidate = generateTest(tested, keyFrame)
tested.add(hash(candidate))
if (isPossiblyPrime(candidate)) arr.push(candidate)
else i--
 ```
 
My reasoning for this is that it'd allow us to continue running enough OpenSSL processes to make full use of all the cores we have available.

Reply:
I just benchmarked with this change applied, and it does appear significantly faster than the original. 

I'm now getting up to 155.5 primes per second, whereas before I was getting between 8 & 30 before.